stream-chat-android-ui-components/src/main/res/values-it/strings_channel_list.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="stream_ui_channel_list_empty">Nessun canale disponibile</string>
+    <string name="stream_ui_channel_list_you">Te</string>
```

Comment:
When generic, the 2nd person pronoun should be `Tu`. I double checked on Telegram in italian.